stacks/web.ts

```diff
@@ -20,7 +20,7 @@ export function Web({ stack }: StackContext) {
         }
       : undefined,
     environment: {
-      // NEXTAUTH_URL: 'http://localhost:6020', // FIXME: how to pass in this URL?
+      NEXTAUTH_URL: 'http://localhost:6020', // FIXME: how to pass in this URL?
```

Comment:
I think someone tried to here https://github.com/nextauthjs/next-auth/issues/969

Reply:
We should have more information in CDK-land though that we could theoretically use to generate this but it's not so simple